Automattic/compare-snapshot-tool
# Add a comment field when rejecting the test

## Changes
Add a comment field when rejecting the test. This field is required and you can only reject a test if a comment is added. Otherwise, the status will not be updated.

1.  A comment section is added below the status. This will display the current comment (if any). 
2. If a failed test is now passing, the user can click on the `Accept` button. 
2.1. The status will be set to PASSED. 
2.2 If there's an existing comment, the comment will be updated. And the text "Previous failure message" will be added at the start. 
4. The auto forward to the next data when clicking **Accept/Reject** button is removed. This is because it could be easier to check the details after changing the comment.

## Testing
Create a CSV file with the following columns: `newUrl`, `oldUrl`, `status`, and `comment`. You can add any URLs in the file.

- Run yarn to install dependencies.
- Run yarn start to see the development build.

Make sure that the functionality is working as expected.
